insights/combiners/selinux.py

```diff
@@ -39,17 +39,16 @@
 BOOT_NOT_ENFORCING = 'selinux_conf_not_enforcing'
 
 
-@combiner(SEStatus, SelinuxConfig,
-          optional=[Grub1Config, Grub1EFIConfig, Grub2Config, Grub2EFIConfig])
+@combiner(SEStatus, SelinuxConfig, GrubConf)
```

Comment:
@xiangce the original logic had grub parsers as optionals, you've made the combiner mandatory here.  The logic still checks for the existence of a grub combiner so I think it should still be optional.

Reply:
Done. Thanks @bfahr